scripts/bootstrap-controllers.sh

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/bash
```

Comment:
Looking through most of the scripts in kubernetes, they always do #!/bin/bash instead of #!/usr/bin/bash.  I believe this this the recommended way for performance reasons and because some distros like MacOS do not install bash in /usr/bin.  I would change this to #!/bin/bash.